CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,15 @@
+Code of Conflict
+----------------
+
+Our development effort is a very personal process compared
+to "traditional" ways of developing software.  Your code and ideas
+behind it will be carefully reviewed, often resulting in critique and
+criticism.  The review will almost always require improvements to the
+code before it can be included.  Know that this happens
+because everyone involved wants to see the best possible solution for
+the overall success of this project.
+
+As a reviewer of code, please strive to keep things civil and focused on
+the technical issues involved.  We are all humans, and frustrations can
+be high on both sides of the process.  Try to keep in mind the immortal
+words of Bill and Ted, "Be excellent to each other."
```

Comment:
There has been an update with our code of conflict.

Please update it and manage it accordingly:
https://github.com/snek-at/front/blob/master/CODE_OF_CONDUCT.md